humanapi/connect-client
# HB-2166: Fix Babel Polyfills

## What did you implement
- Adds core-js babel polyfills
- Includes "manual" polyfills in webpack for non-corejs compatible functions

## How can we verify it
- Include https://cdn.humanapi.co/humanapi-connect-client@2.0.5-beta.1.js in your code
- Try it out with IE11, it should not raise any syntax errors
